pygame 2.0.1 (SDL 2.0.14, Python 3.9.5)
Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "c:\Programming\rl_cooperation\DQN_SpaceCannons\agent.PY", line 311, in <module>
    main(False)
  File "c:\Programming\rl_cooperation\DQN_SpaceCannons\agent.PY", line 105, in main
    env = gym.make("Space_cannons:SpaceCannon-v1")
  File "C:\Programming\rl_cooperation\lib\site-packages\gym\envs\registration.py", line 145, in make
    return registry.make(id, **kwargs)
  File "C:\Programming\rl_cooperation\lib\site-packages\gym\envs\registration.py", line 90, in make
    env = spec.make(**kwargs)
  File "C:\Programming\rl_cooperation\lib\site-packages\gym\envs\registration.py", line 60, in make
    env = cls(**_kwargs)
  File "c:\programming\rl_cooperation\pygame_rl_cooperation_v2\space_cannons\Space_cannons\envs\custom_env.py", line 42, in __init__
    self.action_space.n = 4
AttributeError: 'list' object has no attribute 'n'